src/usr/local/www/diag_ping.php

```diff
@@ -157,6 +168,13 @@
 	array_combine(range(1, MAX_COUNT), range(1, MAX_COUNT))
 ))->setHelp('Select the maximum number of pings.');
 
+$section->addInput(new Form_Select(
+	'wait',
+	'Number of seconds to wait between pings',
+	$wait,
+	array_combine(range(1, MAX_WAIT), range(1, MAX_WAIT))
+))->setHelp('Select the number of seconds to wait between pings.');
+
```

Comment:
There seems no point in the label being the same as the help text. Perhaps simplify the label to "Seconds between pings" of just eliminate the help text ?

Reply:
Changed. I left the help text so that they are consistent. In my opinion consistency trumps removing redundancy in such cases.